compiler/src/Transform.js

```diff
@@ -84,30 +105,58 @@ function Transform(
             console.log(this.query);
         }
         this.separable = query["separable"] || true;
+        this.dependencies = [];
```

Comment:
can you help explain what `dependencies` are for? This ideally should be in the documentation too.

Reply:
Yep! I can put in the Layer API, since the dependencies array is the way to specify whether there is some hierarchical dependency between two (or more) layers. For example, in the hierarchical election example, there is a dependency between the main county dynamic layer and the main state dynamic layer.